packages/components-native/src/StatusLabel/StatusLabel.style.ts

```diff
@@ -1,33 +1,24 @@
 import { StyleSheet } from "react-native";
 import { tokens } from "../utils/design";
 
-const statusLabelIconDiameter = tokens["space-base"] - tokens["space-smaller"]; //12px
-
-const indicatorOffset = tokens["space-smallest"] + tokens["space-minuscule"];
-
-// Needs to be hardcoded to 3 as this shouldn't change with the tokens
-const statusLabelRadius = 3;
-
 export const styles = StyleSheet.create({
   statusLabelRow: {
     flexDirection: "row",
     justifyContent: "flex-end",
+    alignItems: "center",
     flexWrap: "nowrap",
+    flexGrow: 0,
+    flexShrink: 1,
```

Comment:
Not entirely sure if these are required - I'm trying to ensure that the label _doesn't_ get stretched full-width like this:
![image](https://github.com/user-attachments/assets/324ab6ee-befd-451f-9f11-6766cc303466)

But as you can see, that's happening anyways because there's a flex parent on the mobile layout